src/QuarkusExampleCodestartDev.java

```diff
@@ -0,0 +1,141 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//SOURCES WrapperRunner.java
+//DEPS info.picocli:picocli:4.5.0
+//DEPS io.quarkus:quarkus-platform-descriptor-json:1.10.2.Final io.quarkus:quarkus-platform-descriptor-resolver-json:1.10.2.Final
```

Comment:
i'm curious - how is this going to work when we update quarkus - will this stay forward/backwards compatible ?

Reply:
I was thinking we would just update the version here (like we do for the CLI)..

Reply:
Not possible to make it a param anyway...?

Reply:
@maxandersen good to merge?

Reply:
It is possible to parametrize the version. ${someproeprty:1.10.2..}